apps/avifenc.c

```diff
@@ -221,7 +223,8 @@ static void syntaxLong(void)
     printf("    --mini                            : EXPERIMENTAL: Use reduced header if possible (backward-incompatible)\n");
 #endif
     printf("    -l,--lossless                     : Set all defaults to encode losslessly, and emit warnings when settings/input don't allow for it\n");
-    printf("    -d,--depth D                      : Output depth, one of 8, 10 or 12. (JPEG/PNG only; For y4m or stdin, depth is retained)\n");
+    printf("    -d,--depth D                      : Primary item output bit depth per channel. One of: 8/10/12. (JPEG/PNG only; For y4m or stdin, depth is retained)\n");
+    printf("    --depthext Dext Doverlap          : Bit depth extension using Sample Transforms. Allowed [D Dext Doverlap] combinations: [D 0 0]/[8 8 0]/[12 4 0]/[12 8 4]\n");
```

Comment:
I suggest using a comma (,) instead of space between Dexp and Doverlap. See the --crop, --clap, and --clli options below.